Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/PointDrawing.cs

```diff
@@ -233,7 +295,9 @@ public static void Do(int id, Rect dragArea, ref List<Vector3> points, out Matri
                     if (SceneHandles.InCameraOrbitMode)
                         break;
 
-                    if (GUIUtility.hotControl != 0)
+                    var hotControl = GUIUtility.hotControl;
+                    if (hotControl != 0 &&
+                        hotControl != id)
```

Comment:
When releaseOnMouseUp is false, then the current hotControl might still point to our own id.
After we clicked the first time, it won't be set to 0. But we still want to be able to click again